manager/manager.go

```diff
@@ -75,11 +90,24 @@ func NewManager(opts Options) (*Manager, error) {
 	if opts.WriteKeypair == nil {
 		return nil, errors.New("WriteKeypair must be set")
 	}
+	if opts.MaxRequestsPerVolume == 0 {
+		opts.MaxRequestsPerVolume = 3
+	}
+	if opts.MaxRequestsPerVolume < 0 {
+		return nil, errors.New("MaxRequestsPerVolume cannot be less than zero")
+	}
```

Comment:
We may want to consider permitting e.g. `-1` as a value here to disable cleanup altogether. I've opted to omit that option for now though until we actually have someone request it/have a use-case 😄 (e.g. external cleanup controller)